mesheryctl/internal/cli/root/system/start.go

```diff
@@ -83,7 +83,7 @@ var startCmd = &cobra.Command{
 func start() error {
 	if _, err := os.Stat(utils.MesheryFolder); os.IsNotExist(err) {
 		if err := os.Mkdir(utils.MesheryFolder, 0777); err != nil {
-			return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to make %s directory", utils.MesheryFolder)))
+			return handlers.ErrCreateDir(err, utils.MesheryFolder)
```

Comment:
Errors should be local to every package. is there a reason we are using this one from the handler package?

Reply:
@Shreyas220 I didn't want to define an error again if it was already present. I thought it would be redundant. 
If you think that's not the case, I can re-define it in `system`.

Reply:
yes please do it here,
redundancy isn't the issue here, we will be updating these error codes to docs here https://docs.meshery.io/reference/error-codes
every package has its own error codes, making it easy to debug the error if it comes 
using the same error from different packages creates confusion

Reply:
alright @Shreyas220 .
Done it in the latest commit.